### PR TITLE
task 85zrja1u4  Changed the save draft advance request confirmation pop up to new design

### PR DESCRIPTION
**Pull request by CODEX108**
The new pop up design

### CODE REVIEW

File: src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts [hunk 1]
```diff
@@ -239,8 +239,21 @@ export class AddEditAdvanceRequestPage implements OnInit {
   async showAdvanceSummaryPopover() {
     if (this.fg.valid) {
       const advanceSummaryPopover = await this.popoverController.create({
-        component: DraftAdvanceSummaryComponent,
```
Comment (Dimple16) [inline src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts:242]: can you check the usage of this component? if not used anywhere, remove all the files related to `draftadvancesummary` component

File: src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts [hunk 2]
```diff
@@ -239,8 +239,21 @@ export class AddEditAdvanceRequestPage implements OnInit {
...
-        cssClass: 'dialog-popover',
+        component: PopupAlertComponentComponent,
+        componentProps: {
+          title: 'Review',
+          message:
+            'This action will save a draft advance request and will not be submitted to your approvers directly . You need to explicitly submit a draft advance request.',
```
Comment (Dimple16) [inline src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts:246]: directly . -> `directly.`

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

File: src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts (hunk 2 above)
Comment (CODEX108) [reply src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts:246]: ok sure

File: src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts (hunk 1 above)
Comment (CODEX108) [reply src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts:242]: **These files are deleted**



**These files weren't related to any other component except the ```add-edit-advance-request``` files**

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**